src/main/java/org/kohsuke/github/GHPullRequest.java

```diff
@@ -208,11 +202,11 @@ public String getMergeCommitSha() throws IOException {
 
     /**
      * Fully populate the data by retrieving missing data.
-     *
+     * <p>
      * Depending on the original API call where this object is created, it may not contain everything.
      */
     private void populate() throws IOException {
-        if (mergeable_state!=null)    return; // already populated
+        if (mergeable_state != null) return; // already populated
         if (root.isOffline()) {
             return; // cannot populate, will have to live with what we have
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this line start at column 7. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ATrailingCommentCheck)